conf/routes

```diff
@@ -20,7 +20,6 @@ POST        /digital/country                 controllers.DigitalPack.handleCount
 # Checkout
 GET         /checkout                        controllers.Checkout.renderCheckout
 POST        /checkout                        controllers.Checkout.handleCheckout
-GET         /checkout/thankyou               controllers.Checkout.thankyou
```

Comment:
The subscription ID should be enough. We can use that and make a call to Zuora and Salesforce to pull back the relevant information. Most of the groundwork is done for making calls to Zuora and Salesforce so it should not be a great deal of work. 

It's a similar approach to Membership. Happy to pair on this tomorrow or later next week.